charts/daemonset/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: daemonset
+description: A Helm Chart to deploy an arbitrary container as a daemonset.
+version: 0.1.0
+icon: https://raw.githubusercontent.com/snowplow-devops/helm-charts/master/docs/logo/snowplow.png
+home: https://github.com/snowplow-devops/helm-charts
+sources:
+  - https://github.com/snowplow-devops/helm-charts
+maintainers:
+  - name: jbeemster
+    url: https://github.com/jbeemster
+    email: jbeemster@users.noreply.github.com
+keywords:
+  - daemonset
+dependencies:
+  - name: dockerconfigjson
+    version: 0.1.0
```

Comment:
FYI you can add `condition: dockerconfigjson.deploy` to avoid helm requiring a dependency if it's disabled in  values.yaml

Reply:
TIL!  That's a neat trick.

Reply:
Did that for one of the deps - cannot do it for both as I do not have a boolean available I can use.